cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -847,7 +847,7 @@ public:
   {
 // GCC 14 rightfully warns that when a value-initialized array of this struct is copied using memcpy, uninitialized
 // bytes may be accessed. To avoid this, we add a dummy member, so value initialization actually initializes the memory.
-#if defined(_CCCL_COMPILER_GCC) && __GNUC__ == 14
+#if defined(_CCCL_COMPILER_GCC) && (__GNUC__ == 13 || __GNUC__ == 14)
```

Comment:
@griwes said the same when first reviewing this patch. I was just hoping that a future GCC would come with a smarter analysis and this would be temporary workaround. But since it was brought a second time, I will implement it as suggested.